.github/workflows/main.yml

```diff
@@ -232,17 +165,34 @@ jobs:
           tag: ${{ github.ref }}
           overwrite: true
 
+  debian-packaging:
+    name: Ubuntu/Debian packaging (Ubuntu 20.04)
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          fetch-depth: 0
+      - name: install packages
+        run: sudo apt-get update && sudo apt-get install -y build-essential debhelper fakeroot libjsoncpp-dev libcurl4-openssl-dev libtinyxml2-dev libjsoncpp1 libcurl4 libtinyxml2-6a
+      - name: generate changelog
+        run: ./tools/generate_debian_changelog.sh > debian/changelog
+      - name: Build package
+        run: dpkg-buildpackage -us -uc -b
+
   rpm-package:
     name: ${{ matrix.container_name }} (package, non-mavsdk_server)
     runs-on: ubuntu-18.04
     container: mavsdk/mavsdk-${{ matrix.container_name }}
     strategy:
       matrix:
-        container_name: [fedora-32, fedora-33]
+        container_name: [fedora-33]
```

Comment:
Why only Fedora 33? My idea is to always have 2, so always the two versions that are supported.

Reply:
I did not manage to install perl-FindBin on Fedora-32, and eventually gave up :sweat_smile: